.eslintrc.json

```diff
@@ -43,9 +43,24 @@
         "no-useless-escape": "off",
         "no-undef": "off",
         "no-prototype-builtins": "off",
-        "local-rules/no-budibase-imports": "error",
+        "local-rules/no-budibase-imports": "error"
+      }
+    },
+    {
+      "files": ["**/*.spec.ts"],
```

Comment:
Should we use an existing plugin such as `eslint-plugin-jest` instead?

https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-focused-tests.md

Reply:
We could probably also benefit form the other recommended rules

Reply:
I'm in the process of enabling the TypeScript recommended rules in another PR. I'll add the Jest ones in there, too. Good call!